schemas/telemetry/new-profile/new-profile.4.parquetmr.txt

```diff
@@ -0,0 +1,69 @@
+message new-profile {
+    required binary id (UTF8);
+    required binary clientId (UTF8);
+    required group metadata {
+        required int64  Timestamp;
+        optional binary Date (UTF8);
+        optional binary normalizedChannel (UTF8);
+        optional binary geoCountry (UTF8);
+        optional binary geoCity (UTF8);
```

Comment:
Can we also add the "X-PingSender-Version" meta, when available? Would you mind also adding the meta/creationTimestamp?